backup_and_restore/application_backup_and_restore/installing/installing-oadp-kubevirt.adoc

```diff
@@ -0,0 +1,71 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-oadp-kubevirt"]
+// If docs team approves the name, "Configuring OADP with Kubevirt" great but I suspect it will need to be "Configuring OADP with OpenShift Virtualization" or "Configuring OADP with Virtualization" or perhaps... bear with me.. "OADP and OpenShift-Virtualization"
+= Configuring the OpenShift API for Data Protection with {VirtProductName}
+include::_attributes/common-attributes.adoc[]
+:context: installing-oadp-kubevirt
+:installing-oadp-kubevirt:
+:credentials: cloud-credentials
+:provider: kubevirt
+
+toc::[]
+
+You can install the {oadp-first} with {VirtProductName} by installing the OADP Operator and configuring a backup location. Then, you can install the Data Protection Application.
+
+// include::snippets/oadp-mtc-operator.adoc[]
+
+Back up and restore virtual machines by using the xref:../../../backup_and_restore/index.adoc#application-backup-restore-operations-overview[OpenShift API for Data Protection (OADP)].
+
+[NOTE]
+====
+(oadp-full} with {VirtProductName} backup and restore support is limited to:
+* Container Storage Interface (CSI) backups
+* Container Storage Interface (CSI) backups with DataMover
+
+This support excludes:
+* File System backup and restore.
+* Volume Snapshot Backups and Restores.
+
+For more information, see xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-backing-up-applications-restic-doc.adoc#oadp-backing-up-applications-restic-doc[Backing up applications with File System Backup: Kopia or Restic].
+====
```

Comment:
The lists in this admonition do not render correctly in the preview. The attribute also needs to go between two brackets.

Currently:
![image](https://github.com/openshift/openshift-docs/assets/50179998/f28b054c-ffeb-48fb-9e2d-5aebe3c68b57)

This must be fixed prior to merging.

Reply:
```suggestion
[NOTE]
====
{oadp-full} with {VirtProductName} backup and restore support is limited to:

* Container Storage Interface (CSI) backups
* Container Storage Interface (CSI) backups with DataMover

This support excludes:

* File System backup and restore.
* Volume Snapshot Backups and Restores.

For more information, see xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-backing-up-applications-restic-doc.adoc#oadp-backing-up-applications-restic-doc[Backing up applications with File System Backup: Kopia or Restic].
====
```